idea/src/org/jetbrains/kotlin/idea/formatter/BaseKotlinImportLayoutPanel.kt

```diff
@@ -0,0 +1,321 @@
+            return
+        }
+
+        TableUtil.stopEditing(layoutTable)
+        packageTable.removeEntryAt(row)
+
+        val model = layoutTable.model as AbstractTableModel
+        model.fireTableRowsDeleted(row, row)
+
+        if (row >= packageTable.getEntryCount()) {
+            row--
+        }
+
+        if (row >= 0) {
+            layoutTable.setRowSelectionInterval(row, row)
+        }
+    }
+
+    protected fun movePackageUp() {
+        val row = layoutTable.selectedRow
+        if (row < 1) return
+
+        TableUtil.stopEditing(layoutTable)
+        val entry = packageTable.getEntryAt(row)
+        val previousEntry = packageTable.getEntryAt(row - 1)
+        packageTable.setEntryAt(entry, row - 1)
+        packageTable.setEntryAt(previousEntry, row)
+
+        val model = layoutTable.model as AbstractTableModel
+        model.fireTableRowsUpdated(row - 1, row)
+        layoutTable.setRowSelectionInterval(row - 1, row - 1)
+    }
+
+    protected fun movePackageDown() {
+        val row = layoutTable.selectedRow
+        if (row >= packageTable.getEntryCount() - 1) return
+
+        TableUtil.stopEditing(layoutTable)
+        val entry = packageTable.getEntryAt(row)
+        val nextEntry = packageTable.getEntryAt(row + 1)
+        packageTable.setEntryAt(entry, row + 1)
+        packageTable.setEntryAt(nextEntry, row)
+
+        val model = layoutTable.model as AbstractTableModel
+        model.fireTableRowsUpdated(row, row + 1)
+        layoutTable.setRowSelectionInterval(row + 1, row + 1)
+    }
+
+    private fun refreshTableModel(row: Int) {
+        val model = layoutTable.model as AbstractTableModel
+        model.fireTableRowsInserted(row, row)
+        layoutTable.setRowSelectionInterval(row, row)
+        TableUtil.editCellAt(layoutTable, row, 0)
+        val editorComp = layoutTable.editorComponent
+        if (editorComp != null) {
+            IdeFocusManager.getGlobalInstance().doWhenFocusSettlesDown { IdeFocusManager.getGlobalInstance().requestFocus(editorComp, true) }
+        }
+    }
+
+    protected fun resizeColumns() {
+        val packageRenderer: ColoredTableCellRenderer = object : ColoredTableCellRenderer() {
+            override fun customizeCellRenderer(
+                table: JTable,
+                value: Any?,
+                selected: Boolean,
+                hasFocus: Boolean,
+                row: Int,
+                column: Int
+            ) {
+                val entry = packageTable.getEntryAt(row)
+                val attributes = KotlinHighlightingColors.KEYWORD.defaultAttributes
+                append("import", SimpleTextAttributes.fromTextAttributes(attributes))
+                append(" ", SimpleTextAttributes.REGULAR_ATTRIBUTES)
+
+                when (entry) {
+                    KotlinPackageEntry.ALL_OTHER_IMPORTS_ENTRY -> append(
+                        "all other imports",
```

Comment:
@fedochet I couldn't find property with such text, does that mean I should create new one?

Reply:
I am sorry, but it seems like I've misguided you a little bit about what messages should be externalized

It seems like the only message that should be externalized is the `"Import aliases separately"` message

You can add this message in this file and then use `KotlinBundle.message` to obtain it:  https://github.com/JetBrains/kotlin/blob/f494b4ce11b3f950199242d035ee803ac7c59a57/idea/resources/messages/KotlinBundle.properties

The messages that you've fixed actually do not require fixing, because 
1) `setButtonComparator` expects buttons names, which seems to be hardcoded, and replacing them with even slightly different messages will lead to broken (or just default) sorting 
2) In the case of `import all other imports` or `import aliases` it would be better to have them in English, as it is more a code example then a UI message

I am sorry that you have to revert some of your changes back, I should have paid more attention to the code 

If it is troublesome for you to revert those changes, please let me know here and I will just amend them into your commits myself before the merge (your authorship of the code will remain intact, but I will merge the PR by hand, not on the Github)

Of course, you are welcome to fix the problem yourself if you like to

Reply:
@fedochet I have sent new commit that should address your last comment, would you mind to look at the PR again?

Reply:
@gcx11 sure! I will take a look tomorrow